README.md

```diff
@@ -65,6 +65,47 @@ After you have a configuration that resembles the following:
 
 `base64` encode the JSON and provide it to envbuilder as the `DOCKER_CONFIG_BASE64` environment variable.
 
+Alternatively, if running `envbuilder` in Kubernetes, you can create an `ImagePullSecret` and
+pass it into the pod as a volume mount:
+
+```shell
+# Artifactory example
```

Comment:
Technically this would work with any registry, so maybe we mention up above in a Note that it does